terragraph_planner/los/helper.py

```diff
@@ -546,17 +547,19 @@ def compute_los_batch(
     if use_ellipsoidal_los_model:
         los_validator = EllipsoidalLOSValidator(
             surface_elevation,
-            max_los_distance,
-            min_los_distance,
+            max_los_distance * (1 + DISTANCE_TOLERANCE_PERCENT),
+            min_los_distance * (1 - DISTANCE_TOLERANCE_PERCENT),
+            ELE_SCAN_ANGLE_LIMIT + ELE_SCAN_ANGLE_TOLERANCE,
             carrier_frequency,
             exclusion_zones,
             los_confidence_threshold,
         )
     else:
         los_validator = CylindricalLOSValidator(
             surface_elevation,
-            max_los_distance,
-            min_los_distance,
+            max_los_distance * (1 + DISTANCE_TOLERANCE_PERCENT),
```

Comment:
You added a tolerance here and check the distance again when adding links to topology because it's assumed the haversine distance is assumed to be more accurate, but is that always the case? I'm not sure about it, but just wondering. But given that we use haversine distance in reporting, I'm good with it.

Reply:
I don't know if it's more accurate. It's more about being self-consistent because we use link.distance and link.el_dev everywhere else (eg link budgets).

Reply:
Yes. I think so. It's good to keep consistent.